src/openms/source/CHEMISTRY/EnzymaticDigestion.cpp

```diff
@@ -261,20 +156,20 @@ namespace OpenMS
     {
       return true; // we don't care about terminal ends
     }
-    else
-    { // either SPEC_SEMI or SPEC_FULL
+    else // either SPEC_SEMI or SPEC_FULL
+    {
       bool spec_c = false, spec_n = false;
 
+      std::vector<Size> pep_positions = tokenise(protein.toUnmodifiedString());
       // test each end
       if (pep_pos == 0 ||
           (pep_pos == 1 && protein.getResidue((Size)0).getOneLetterCode() == "M") ||
```

Comment:
This is the methionine rule at the protein start and which assumes the presence of cellular enzymes.
Question is how this should be supported? Easiest would be to just create a parameter for it (bool allow_methionine_cleavage_at_protein_start).